### PR TITLE
minor fix on typo

### PR DESCRIPTION
**Pull request by tuxerrante**
Explicit memory settings for new users. Little typo.

### CODE REVIEW

File: foundational/PredictableDemands/README.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ To access the PersistentVolume used in this demo, let's mount a local directory
 
 [source, bash]
 ----
-minikube start --mount --mount-string="$(pwd)/logs:/example"
+minikube start --mount --mount-string="$(pwd)/logs:/example" --memory 2g
```
Comment (rhuss) [inline foundational/PredictableDemands/README.adoc:13]: Particular reason why you do need the `--memory` option here ?

**Review by rhuss — COMMENTED** [review]
Thanks ! Looks good but I have a question about the memory option that you have introduced

**Comment by tuxerrante** [comment]
I've just added it to make it more explicit how to increase it when someone
is copy pasting fast the code.
In my case for knative I had to increase it to 8g.
Nothing special :)

Il Sab 20 Feb 2021, 17:18 Roland Huß <notifications@github.com> ha scritto:

**Comment by rhuss** [comment]
Oops, the PR seems to be messed up quite a bit. Could you briefly summarize the typo that you have fixed, so I just would add it manually ? thx!

**Comment by tuxerrante** [comment]
I'd say it was just a fix from "the" to "to" at line 184
https://github.com/k8spatterns/examples/pull/20/commits/cad4b5fabe5d73f22816c63dcabfc04e77306b31

And the "--memory" argument mentioned above
thanks

**Comment by rhuss** [comment]
Thanks ! I added the change manually, so closing this PR

**Pull request closed without merging**